pkgs/development/tools/global-platform-pro/default.nix

```diff
@@ -62,7 +32,7 @@ stdenv.mkDerivation rec {
     '';
     outputHashAlgo = "sha256";
     outputHashMode = "recursive";
-    outputHash = "15bbi7z9v601all9vr2azh8nk8rpz2vd91yvvw8id6birnbhn3if";
+    outputHash = "1qwgvz6l5wia8q5824c9f3iwyapfskljhqf1z09fw6jjj1jy3b15";
```

Comment:
I get a different checksum here:

```
shrinking RPATHs of ELF executables and libraries in /nix/store/2fzzdg9apid99j7fz9x121snrd73r533-global-platform-pro-18.09.14-deps
strip is /nix/store/vcc4svb8gy29g4pam2zja6llkbcwsyiq-binutils-2.30/bin/strip
patching script interpreter paths in /nix/store/2fzzdg9apid99j7fz9x121snrd73r533-global-platform-pro-18.09.14-deps
checking for references to /build in /nix/store/2fzzdg9apid99j7fz9x121snrd73r533-global-platform-pro-18.09.14-deps...
fixed-output derivation produced path '/nix/store/say459vw54fsqhzxknnkdqmwl178dhvs-global-platform-pro-18.09.14-deps' with sha256 hash '1gjjd02lzc44f2bbyr0vy25cp2l8w6cg3k00y8y21g1wj45dirb0' instead of the expected hash '1qwgvz6l5wia8q5824c9f3iwyapfskljhqf1z09fw6jjj1jy3b15'
error: build of '/nix/store/3h1ahmc02xvd0yiiz4935xjq4zs047cg-global-platform-pro-18.09.14-deps.drv' on 'ssh://nix@inspector.r' failed: fixed-output derivation produced path '/nix/store/say459vw54fsqhzxknnkdqmwl178dhvs-global-platform-pro-18.09.14-deps' with sha256 hash '1gjjd02lzc44f2bbyr0vy25cp2l8w6cg3k00y8y21g1wj45dirb0' instead of the expected hash '1qwgvz6l5wia8q5824c9f3iwyapfskljhqf1z09fw6jjj1jy3b15'
```

Reply:
Hmm… On a different computer on a different network I'm getting the same hash, and ofborg appears to have successfully built… are you on x86-64-linux / did you have some transient error that could have made [that loop](https://github.com/NixOS/nixpkgs/pull/48421/files/a55eb78a0d56c7971e8b5e0d6ee2496fbe33508a#diff-3f80555b27d7cb249ab01d104b052e6eL53) fail?